ui/src/actions/workflow.js

```diff
@@ -1,4 +1,7 @@
-import { sendSubmitWorkflowParameters } from '../api/workflow';
+import {
+  sendSubmitWorkflowParameters,
+  sendUpdatedGphlWorkflowParameters,
+} from '../api/workflow';
```

Comment:
If we harmonize the methods on the back end we might be able to get rid of some of this extra logic, so that we for instance don't need ` sendSubmitWorkflowParameters` and ` sendUpdatedGphlWorkflowParameters`